pkgs/development/python-modules/pip-tools/default.nix

```diff
@@ -24,14 +21,23 @@ buildPythonPackage rec {
     sha256 = "992d968df6f1a19d4d37c53b68b3d4b601b894fb3ee0926d1fa762ebc7c7e9e9";
   };
 
-  LC_ALL = "en_US.UTF-8";
-  checkInputs = [ pytest git glibcLocales mock pytest-xdist ];
-  propagatedBuildInputs = [ pip click setuptools-scm pep517 ];
+  checkInputs = [
+    pytestCheckHook
+    pytest-xdist
+  ];
 
-  checkPhase = ''
-    export HOME=$(mktemp -d) VIRTUAL_ENV=1
-    py.test -m "not network"
-  '';
+  propagatedBuildInputs = [
+    click
+    pep517
+    pip
+    setuptools-scm
```

Comment:
Looks like that `setuptools-scm` could be [moved to `nativeBuildInputs`](https://github.com/jazzband/pip-tools/blob/master/setup.cfg#L29).